src/z64.h

```diff
@@ -60,13 +63,48 @@
 extern SDL_Surface *sdl_Screen;
 extern int screen_width, screen_height;
 
-typedef uint64_t UINT64;
-typedef int64_t INT64;
-typedef uint32_t UINT32;
-typedef int32_t INT32;
-typedef uint16_t UINT16;
-typedef int16_t INT16;
-typedef uint8_t UINT8;
-typedef int8_t INT8;
 typedef unsigned int offs_t;
 
+/*
+ * fast fix to missing <stdint.h> report
+ * These defs are not really as universal as the cxd4/rcp/my_types.h ones.
+ */
+#if (SCHAR_MIN <= -128 && SCHAR_MAX >= +127)
+typedef signed char INT8;
+typedef unsigned char UINT8;
+#else
+typedef signed int INT8;
+typedef unsigned int UINT8;
+#endif
+
+#if (SHORT_MIN <= -32768 && SHORT_MAX >= +32767)
```

Comment:
SHORT_ probably was a typo for SHRT_ on the cross-reference I did.

Did not think it would be a compile error though since that's a pre-processor thing.  Go ahead and change it to SHRT_MIN and _MAX, or wait for me to get home on a non-busy day and I'll have time to update the PR myself.

@Frank74,
because stdint.h sucks and shouldn't need to be needed.
The limits.h method is more portable.

The method you described was what I did with Project64 though since zilmar prefers stdint.h but frankly in this case I don't see the necessity for an #ifdef when we can use the equally if not better functioning ANSI path.